README.md

```diff
@@ -1,46 +1,37 @@
-# Getting Started with Create React App
+<h1 align="center">Space Burger</h1>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">Web application for ordering in a burger shop using the burger constructor</div>
 
-## Available Scripts
+![Tech logos](https://i.ibb.co/DVFj8PL/tech-icons.jpg)
 
-In the project directory, you can run:
+![App screenshot](./readme_src/screenshot-app.PNG)
 
-### `yarn start`
+## What is this web application for? 🤷‍♀️
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+It is an interactive application that allows the user to conveniently assemble their own special burger using the constructor.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+Also for registered users, you can view the order history. The user can see the order feed - a list of burgers that are being prepared right now or are already ready. This functionality is available live thanks to the web socket connection.
```

Comment:
Also, as a registered user, you can view the order history. In addition, the user can see the order feed. There is a list of burgers that are being prepared right now or which have already been cooked. This functionality is available in real-time due to the web socket connection.